great_expectations/util.py

```diff
@@ -1926,12 +1930,26 @@ def get_context(
 
     # Second, check for which type of local
     # Prioritize FileDataContext but default to EphemeralDataContext if no context_root_dir
-    if context_root_dir or not project_config:
+    if not context_root_dir:
+        try:
+            context_root_dir = FileDataContext.find_context_root_dir()
+        except gx_exceptions.ConfigNotFoundError:
+            logger.info("Could not find local context root directory")
```

Comment:
❤️ - side note, I've always wondered where this actually looks. Does that information get logged anywhere?

Reply:
yeah `find_context_root_dir` should emit log statements I believe!